src/RoadMarking/RoadMarking.types.tsx

```diff
@@ -0,0 +1,23 @@
+export type RoadMarkingProps = {
+  /**
+   * The points representing the path of the marking [x0, y0, z0, x1, y1, z1, ...]
```

Comment:
Do we need the z-coordinates? This should always be a 2D component

Reply:
Removed, only x and y now.